### PR TITLE
Feature/migrate warehouse next entity

### PR DESCRIPTION
**Pull request by chdietz**
There are some entity steps that do not have their own IT tests but the end to end job IT tests, there are 2, cover > 90% coverage.
Will start working on those, while exam schema refactoring is in progress.

### CODE REVIEW

File: migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/step/WarehouseToStagingStepsConfig.java [hunk 1]
```diff
@@ -76,6 +80,59 @@ public Step stageOrganizationsStep() {
...
+
+    @Bean
+    public Step stageExamStep() {
+        final SqlCopyBuilder sqlsBuilder = new SqlCopyBuilder(warehouseToStagingSqlConfiguration.getEntities());
+
+        sqlsBuilder.addNext("iab_exam_student")
+                .addNext("iab_exam")
+                .addNext("iab_exam_item")
+                .addNext("iab_exam_available_accommodation")
+                .addNext("exam_student")
```
Comment (agorina) [inline migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/step/WarehouseToStagingStepsConfig.java]: You have the steps, but do not have the test data set up. Is my understanding correct?

File: migrate-reporting/src/test/java/org/opentestsystem/rdw/ingest/migrate/reporting/ReportingMigrateJobIT.java [hunk 2]
```diff
@@ -111,7 +106,8 @@ public void itShouldUpsertAndDeleteEntities() throws Exception {
 
         tableTestCounts.add(new TableTestCountHelper(jdbcTemplate, "reporting_test.student", -1, "id in (-89)"));
         tableTestCounts.add(new TableTestCountHelper(jdbcTemplate, "reporting_test.student", 3, "id in (-88, -87, -86)"));
-        tableTestCounts.add(new TableTestCountHelper(jdbcTemplate, "reporting_test.student_ethnicity", 4, "student_id in (-89, -88, -87, -86)"));
+        // todo: there is an issue with delete entities in stage to reporting and no integrity on student_id
```
Comment (agorina) [inline migrate-reporting/src/test/java/org/opentestsystem/rdw/ingest/migrate/reporting/ReportingMigrateJobIT.java]: Is this TODO valid? I thought we have resolved it.

File: migrate-reporting/src/test/resources/Warehouse-Temp-MigrateEntitiesFromStagingForDeleteSetup.sql [hunk 3]
```diff
@@ -7,27 +7,27 @@
...
- ------------------------------------------  Exams ---------------------------------------------------------------------------------------------
-UPDATE staging_test.staging_exam SET deleted = 1 WHERE id = -88;
-DELETE from staging_test.staging_exam_student WHERE id = -18;
-DELETE from staging_test.staging_exam_available_accommodation  WHERE exam_id = -88;
-DELETE from staging_test.staging_exam_item  WHERE exam_id = -88;
+-- ------------------------------------------  Exams ---------------------------------------------------------------------------------------------
+-- UPDATE staging_test.staging_exam SET deleted = 1 WHERE id = -88;
+-- DELETE from staging_test.staging_exam_student WHERE id = -18;
+-- DELETE from staging_test.staging_exam_available_accommodation  WHERE exam_id = -88;
+-- DELETE from staging_test.staging_exam_item  WHERE exam_id = -88;
```
Comment (agorina) [inline migrate-reporting/src/test/resources/Warehouse-Temp-MigrateEntitiesFromStagingForDeleteSetup.sql]: Are you going to delete this file?

File: migrate-reporting/src/test/java/org/opentestsystem/rdw/ingest/migrate/reporting/ReportingMigrateJobIT.java (hunk 2 above)
Comment (chdietz) [inline migrate-reporting/src/test/java/org/opentestsystem/rdw/ingest/migrate/reporting/ReportingMigrateJobIT.java]: ah, no i fixed it, forgot to remove the comment.

File: migrate-reporting/src/test/resources/Warehouse-Temp-MigrateEntitiesFromStagingForDeleteSetup.sql (hunk 3 above)
Comment (chdietz) [inline migrate-reporting/src/test/resources/Warehouse-Temp-MigrateEntitiesFromStagingForDeleteSetup.sql]: yes

**Pull request merged**